activiti-cloud-services-query/activiti-cloud-services-query-graphql-ws/src/main/java/org/activiti/cloud/services/query/qraphql/ws/config/GraphQLSubscriptionsSchemaConfiguration.java

```diff
@@ -25,16 +25,22 @@
 import org.activiti.cloud.services.query.qraphql.ws.schema.GraphQLSubscriptionSchemaBuilder;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
 
 @Configuration
 @EnableActivitiGraphQLQueryService
+@ConditionalOnProperty(name="spring.activiti.cloud.services.notifications.gateway.enabled", matchIfMissing = true)
+@ConditionalOnExpression("${spring.activiti.cloud.services.query.graphql.enabled}==null or ${spring.activiti.cloud.services.query.graphql.enabled}")
```

Comment:
What about create a [meta-annotation](http://www.logicbig.com/tutorials/spring-framework/spring-web-mvc/meta-annotation/) to avoid repeating this too annotations?

Reply:
Another question, isn't `@ConditionalOnExpression("${spring.activiti.cloud.services.query.graphql.enabled}==null or ${spring.activiti.cloud.services.query.graphql.enabled}")` equivalent to `@ConditionalOnProperty(name="spring.activiti.cloud.services.query.graphql.enabled", matchIfMissing = true)`? Is there any specific reason for using different approaches for `spring.activiti.cloud.services.query.graphql.enabled` and `spring.activiti.cloud.services.notifications.gateway.enabled`?